modules/porous_flow/include/materials/PorousFlowBrine.h

```diff
@@ -50,4 +50,7 @@ class PorousFlowBrine : public PorousFlowFluidPropertiesBase
 
   /// NaCl mass fraction at the qps or nodes
   const VariableValue & _xnacl;
+
+  /// Flag to denote whether NaCl mass fraction is a nonlinear variable
```

Comment:
should you add in docstring for the xnacl parameter that the xnacl needs to be listed in. the Dictator as a porous_flow_variable or is this explained somewhere else?

Reply:
Good question. I'll add something in the documentation (just looking and it looks like the documentation for this material is pretty sparse).